satellite/referral/offersweb/pages/index.html

```diff
@@ -0,0 +1,24 @@
+{{define "head"}}
```

Comment:
Just an idea and would like to have some input from others for this. Should we move the templates into the `/web` folder and then create a config for the static assets folder path?

Reply:
I like that idea!

Reply:
@stefanbenten as far as structuring paths, we currently have ```satellite/referral/offersweb/server.go```
is there a reason it can't be ```satellite/referral/server.go``` just trying to get a feel for our do's and don'ts for path structuring.

Reply:
The idea of having a `web` was to separate server static files (`html`, `js`, etc) from the rest of `go` code